common/app/common/configuration.scala

```diff
@@ -20,64 +20,73 @@ import scala.util.{Failure, Success, Try}
 
 class BadConfigurationException(msg: String) extends RuntimeException(msg)
 
-case class InstallVars(stack: String, app: String, guStage: String, awsRegion: String, configBucket: String)
+object InstallVars {
```

Comment:
I decided against moving this to another file as I didn't want to add any more complication to the file structure. Lemme know if you think that's a bad idea